modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/internal/workflow/test/DLFileEntryWorkflowHandlerTest.java

```diff
@@ -61,70 +61,104 @@ public class DLFileEntryWorkflowHandlerTest {
 	@Before
 	public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
+
+		_serviceContext = ServiceContextTestUtil.getServiceContext(
+			_group.getGroupId(), TestPropsValues.getUserId());
+
+		_folder = _getSingleApproverFolder();
```

Comment:
@AliciaGarciaGarcia, all looks good :). Here I would just pass the serviceContext as parameter to explicitly note that an order should be followed.

What do you think?

Reply:
great idea, I'll do it